test/dotnet-add-p2p.Tests/GivenDotnetAddP2P.cs

```diff
@@ -13,45 +12,13 @@ namespace Microsoft.DotNet.Cli.Add.P2P.Tests
 {
     public class GivenDotnetAddP2P : TestBase
     {
+        const string TestGroup = "NonRestoredTestProjects";
+        const string ProjectName = "DotnetAddP2PProjects";
```

Comment:
I really don't like this growing pattern of "throw all my pet projects in a single test asset". It makes it very difficult to understand the intent of a test asset, to reuse them across scenarios. It also makes for messier unit tests since we specify the actual asset under test in two places.